### PR TITLE
Updated examples to work with smaller screen better

### PR DESCRIPTION
**Pull request by makermelissa**
Now that I have a smaller screen that I can experiment with, I have updated the examples so that text and images are centered on both the large and small screen.

### CODE REVIEW

**Comment by makermelissa** [comment]
Could somebody please do a review on this? I would like it merged before I finish my guide. Thanks

**Comment by makermelissa** [comment]
Summary of changes:
Images and text are now centered and fit on both resolutions
Fixed Coloration of 24-bit BMP example
Fixed 16-bit BMP decoding in BMP example

**Comment by siddacious** [comment]
👀

File: examples/ra8875_bmptest.py [hunk 1]
```diff
@@ -27,6 +27,9 @@
 display.init()
 display.fill(WHITE)
 
+def color555(self, rgb):
```
Comment (siddacious) [inline examples/ra8875_bmptest.py]: I think this should be renamed something like `cast_to_565` or something similar. Compared to `color565` the name suggests that it's converting _to_ `555`. 

Alternatively you could import `color565` as `color_565_from_24bpp` and then make your own `color565` that does the appropriate conversion based on bpp? Not a deal breaker either way

Comment (makermelissa) [reply examples/ra8875_bmptest.py]: I just went with convert_555_to_565()

**Pull request merged**